### PR TITLE
Convert v1 components to v3

### PR DESCRIPTION
**Pull request by cosu419**
## Summary
The newly generated "Find a Representative" application has been merged to main, but it is not enabled in production. This PR updates UI components to "uswds" (v3) standard, in addition to addressing styling regression issues. 

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/69322

## Testing done

None. We're developing the 'Find a Representative' app based on approved wireframes but no user-testing has been conducted yet. User-testing is happening in parallel with the development of the app. Once UX Research comes back with the user-testing results, we'll update the app accordingly and write tests.

In short, we don't want to write tests for an implementation that may very well change based on user-testing.

### CODE REVIEW

**Review by holdenhinkle — APPROVED** [review]
LGTM!

**Review by powellkerry — APPROVED** [review]

File: src/applications/representative/components/search/SearchControls.jsx [hunk 1]
```diff
@@ -183,29 +147,21 @@ const SearchControls = props => {
...
+            hint={null}
+            label={
+              representativeType === 'Veteran Service Organization (VSO)'
+                ? 'Organization Name'
+                : 'Representative Name'
+            }
+            message-aria-describedby="Text input for organization or representative name"
+            name="Organization or Representative Name"
+            onChange={handleRepOrganizationChange}
+            // onBlur={handleRepOrganizationBlur}
```
Comment (jvcAdHoc) [inline src/applications/representative/components/search/SearchControls.jsx:160]: Was this intentionally left in?

Comment (cosu419) [reply src/applications/representative/components/search/SearchControls.jsx:160]: Good call, unintentional but I may be re-introducing it (will address in my next PR).
